### ISSUE TITLE
Space between pullers and rope

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/gravity-force-lab-basics#19]
When force is large there is some force between pullers and rope.

**Comment by jessegreenberg** [comment]
![rope](https://user-images.githubusercontent.com/6396244/29795004-94d4a196-8c07-11e7-8821-ad1130881728.PNG)

**Comment by jessegreenberg** [comment]
Increasing  the rope length also pushes the puller away. Wondering why this isn't an issue in coulombs-law.

**Comment by jessegreenberg** [comment]
In ISLCPullerNode, this code
```js
      if ( _.includes( pullImages, images[ i ].children[ 1 ].image ) ) {
        images[ i ].right += i / 5;
      }
```
fixes this issue in coulombs-law, but i/5 is not enough for gravity-force-lab-basics.

**Comment by jessegreenberg** [comment]
Instead of i / 5, could it be a proportion of the image's width?

**Comment by jessegreenberg** [comment]
That seems to work better, but Ill create an issue in inverse-square-law-common.

**Comment by jessegreenberg** [comment]
Fixed in phetsims/inverse-square-law-common#18, closing
![capture](https://user-images.githubusercontent.com/6396244/29796286-85030c66-8c0d-11e7-8c62-5788ffc065c1.PNG)
